### DISCUSSION TITLE
BetterDiscord plugin

### DISCUSSION THREAD

**Discussion by Pendred** [victornpb/undiscord#310] [3. Ideas]
Would it be possible to create a plugin compatible with the BetterDiscord program that would allow this script to be used within the Discord application instead of having to open a browser? Would be very cool and make my life easier. Thanks in advance!

**Comment by victornpb** [comment]
Yes I’m working on it

**Reply by victornpb** [reply]
Work in progress https://github.com/victornpb/undiscord/pull/357
